test/jasmine/tests/select_test.js

```diff
@@ -2192,6 +2192,75 @@ describe('Test select box and lasso per trace:', function() {
         .then(done);
     });
 
+    it('@noCI @flaky should work for funnel traces', function(done) {
```

Comment:
Funnel traces should probably have special `selectPoints` logic. Instead of selecting bars by their mid-position pt at full height, we should probably use their mid-position pt at **mid** height.

Reply:
Good call. Addressed in eea4a72a695e238832506803a2358360cce55c69 and updated the PR codepen.

Reply:
I like it. Thanks!